webrecorder/webrecorder/models/collection.py

```diff
@@ -751,20 +750,26 @@ def has_cdxj(self):
         coll_cdxj_key = self.COLL_CDXJ_KEY.format(coll=self.my_id)
         return self.redis.exists(coll_cdxj_key)
 
+    def reset_cdxj_ttl(self, key=None):
+        if not key:
+            key = self.COLL_CDXJ_KEY.format(coll=self.my_id)
+        if self.COLL_CDXJ_TTL > 0:
+            self.redis.expire(key, self.COLL_CDXJ_TTL)
+            return True
+        return False
+
     def sync_coll_index(self, exists=False, do_async=False):
         coll_cdxj_key = self.COLL_CDXJ_KEY.format(coll=self.my_id)
         if exists != self.redis.exists(coll_cdxj_key):
-            if self.COLL_CDXJ_TTL > 0:
-                self.redis.expire(coll_cdxj_key, self.COLL_CDXJ_TTL)
+            self.reset_cdxj_ttl()
```

Comment:
```suggestion
            self.reset_cdxj_ttl(coll_cdxj_key)
```
Should pass the existing key here, right?